### DISCUSSION TITLE
Can a 'Light' appearance be added to the options?

### DISCUSSION THREAD

**Discussion by ColumbusAI** [Vali-98/ChatterUI#80] [Ideas]
I know a lot of design UIs have moved towards a dark design but I really do think there are many of us that still like a light appearance or the option to have a light appearance. Can this be added to the app?

**Comment by Vali-98** [comment]
This is planned, and with a very rough experimental version when enabling dev mode and toggling it in the colors settings.

There is a lot to refactor under the hood for managing overall app styling, so it will be a while before this can be done cleanly and maintainably.

**Reply by ColumbusAI** [reply]
hey @Vali-98 thanks for the update and detail -- much appreciated!
